app/Domain/Calendar/Repositories/Calendar.php

```diff
@@ -49,23 +41,27 @@ class Calendar extends RepositoryCore
         );
 
         /**
-         * __construct - get database connection
+         * Class constructor.
          *
-         * @access public
+         * @param DbCore $db The DbCore object.
+         * @param LanguageCore $language The LanguageCore object.
+         * @param DateTimeHelper $dateTimeHelper The DateTimeHelper object.
+         * @param Environment $config The Environment object.
+         * @return void
          */
-        public function __construct(DbCore $db, LanguageCore $language, DateTimeHelper $dateTimeHelper)
+        public function __construct(
+            private DbCore $db,
+            private LanguageCore $language,
+            private DateTimeHelper $dateTimeHelper,
+            private Environment $config)
         {
             $this->db = $db;
             $this->language = $language;
             $this->entity = "calendar";
             $this->dateTimeHelper = $dateTimeHelper;
+            $this->config = $config;
```

Comment:
You don't need any of these when you use the new constructor format. Also, $this->config doesn't exist.